app/controllers/items_controller.rb

```diff
@@ -18,17 +19,34 @@ def new
   def create
     item = Item.new(item_params)
     item.seller_id = current_user.id
-    @presenter = NewItemPresenter.new(item)
-    if @presenter.item.save
+    if item.valid?
+      item.save
+      if auctioned?(item)
```

Comment:
You misinterpreted the original comment here. I want to be able to do this: `item.auctioned?`